CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Made sure schemas of an unrecognized type raise a SchemaParseError (not Name error)
 
 ### Changed
+- Attached failed `uri` or `pathname`  to read errors for more meaning
```

Comment:
Sorry, that should be

    # [Unreleased]

    # Changed
    - Attached failed `uri` or `pathname`  to read errors for more meaning

    # [2.6.1] - 2016-02-26

(right now you've put your change under the 2.6.1 release, which has already been published